### PR TITLE
Konstantinos / [FE Deriv.com squad] Africa Award and Latam Award Improvement

### PR DESCRIPTION
**Pull request by konstantinosG-deriv**
Changes:

-   ...

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [x] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: src/pages/why-choose-us/_africa-forex-award.tsx [hunk 1]
```diff
@@ -1,45 +1,166 @@
...
+        padding: 0 50px 40px 50px;
+    }
+`
+const StyledFlexMobile = styled(Flex)`
+    /* padding: 80px 120px 160px 120px;
+    
+    @media (max-width: 1200px) {
+        padding: 0 50px 40px 50px;
+    } */
+`
```
Comment (michio-rework) [inline src/pages/why-choose-us/_africa-forex-award.tsx:81]: do we need this component? 
the whole style is commented.

File: src/pages/why-choose-us/_africa-forex-award.tsx [hunk 2]
```diff
@@ -1,45 +1,166 @@
...
 
-const ImgWrapper = styled.div`
+const AwardImageWrapper = styled.div`
     position: relative;
-    max-height: 1050px;
-    max-width: 1680px;
-    left: 50%;
-    transform: translateX(-50%);
+
+    /* right: -5.5%; */
```
Comment (michio-rework) [inline src/pages/why-choose-us/_africa-forex-award.tsx:27]: please remove this commented line, thanks thanks.

File: src/pages/why-choose-us/_africa-forex-award.tsx [hunk 3]
```diff
@@ -1,45 +1,166 @@
...
-        object-fit: contain !important;
+const AwardImageWrapperMobile = styled.div`
+    z-index: 2;
+    padding: 36px 18px;
+    max-width: 328px;
+    background: #ffffff;
+
+    /* box-shadow: 0px 32px 64px -12px rgba(14, 14, 14, .14); */
+
+    /* box-shadow: 0px 25px 50px -9.5px rgba(14, 14, 14, 0.14); */
```
Comment (michio-rework) [inline src/pages/why-choose-us/_africa-forex-award.tsx:64]: please remove these commented lines, thanks thanks.

**Review by michio-rework — CHANGES_REQUESTED** [review]
- The responsive mobile version is not the same as design. please update it thanks. 
- please remove `withIntl` from these 2 components, we don't need it there.
- the right side card for the award badge needs shadow based on the design, please update it accordingly.

File: src/pages/why-choose-us/_africa-forex-award.tsx (hunk 1 above)
Comment (konstantinosG-deriv) [reply src/pages/why-choose-us/_africa-forex-award.tsx:81]: Forgot about that, thanks

**Review by michio-rework — DISMISSED** [review]

**Review by habib-deriv — APPROVED** [review]

**Pull request merged**